conf/default.yaml

```diff
@@ -9,6 +9,9 @@ server:
   address: '0.0.0.0'
 routePrefix: "{{ ENV.ROUTE_PREFIX || ENV.PATH_PREFIX || '' }}"
 plugins:
+  hapi-config-route:
+    _enabled: '{{ENV.CONFIG_KEY !== undefined}}'
```

Comment:
Don't we have a helper for this?

Reply:
We do but it is not being used in rapptor currently.